modules/Leads/views/view.convertlead.php

```diff
@@ -509,6 +510,23 @@ protected function handleSave()
             $lead->converted = '1';
             $lead->in_workflow = true;
             $lead->save();
+
+            // handle copying photo file, if present:
+            if (!empty($lead->id) && !empty($lead->photo) &&
+                !empty($beans['Contacts']->id) && !empty($beans['Contacts']->photo)) {
+                $bCopied = false;
+                if (($lead->photo == $beans['Contacts']->photo) && is_readable('upload/' . $lead->id . '_photo')) {
+                    $bCopied = copy('upload/' . $lead->id . '_photo',
+                                   'upload/' . $beans['Contacts']->id . '_photo');
+                }
+                if ($bCopied) {
+                    $beans['Contacts']->photo = $lead->photo;
+                    $beans['Contacts']->save();
+                } else {
+                    $GLOBALS['log']->warning('Lead conversion: failed copying upload/' .
```

Comment:
```suggestion
                    LoggerManager::getLogger()->warning('Lead conversion: failed copying upload/');
```

Reply:
Ok. Let me just ask you a few question for my education on this...

1. Is this the general rule now - always use the new syntax?
2. And whenever I touch a file that has the old syntax, change all occurrences to the new?
3. I assume we don't just find-and-replace this all over the code, because in some places it might break?

Reply:
1. I don't think it's actually in the style guide but I would say yes. We should keep it consistent.
2. You wouldn't necessarily need to change everywhere in the file unless you really wanted to. In general, I would say you only really need to do so for the functions you touch (Same for PSR-2, etc).
3. I highly doubt find and replace would cause any issue, these are functionally identical. It's simply that nobody has got around to doing it yet afaik.